res/database/Default/database_programmes.xml

```diff
@@ -34646,6 +34717,7 @@ Zawiera wiele kalamburów i [3|Full] w nagrodzonym Oscarem humorze.</pl>
 			<title>
 				<de>Hannah - Der Kinofilm</de>
 				<en>Hannah</en>
+				<pl>Baletnica Anna</pl>
```

Comment:
... or did you put the original titles in the "spoofed" title tag.

Reply:
Changed.